src/Model/ResolveTokenResult.php

```diff
@@ -17,6 +18,7 @@ public static function fromResponse(array $data): self
     {
         return new self(
             $data['subscriptionId'],
+            $data['subscriptionData'],
```

Comment:
Potreboval by som este nejake data naviac co pouzivame pre azure, povodne sme mali toho klienta c&p v connection https://github.com/keboola/connection/pull/4772/files#diff-3e219885e47031312bdd6aa5508cb972f58a7a0d0ffaa42f422716476d0b9ea9L33

Reply:
Zajimavy, ze to tam pred tim nebylo :thinking: Nevis jak to fungovalo??

Reply:
Aha, vy jste v Connection meli nejakeho vlastniho Billing API clienta, ze?

Reply:
Jj bol to c&p, neviem ci to Tomas vtedy odniekal vykopiroval a pridal tom tam alebo to tam niekde bolo